test/shopify-cli/api_test.rb

```diff
@@ -56,5 +62,37 @@ def test_raises_error_with_invalid_url
         api.query('api/mutation')
       end
     end
+
+    def test_load_query_can_load_project_type_queries
+      new_api = TestAPI.new(ctx: Context.new, token: 'blah', url: 'alsoblah')
+      ShopifyCli::Project.expects(:current_project_type).returns(:fake)
+
+      expected_path = File.join(ShopifyCli::ROOT, 'lib', 'project_types', 'fake', 'graphql', 'my_query.graphql')
+      File.expects(:exist?).with(expected_path).returns(true)
+      File.expects(:read).with(expected_path).returns('content')
+
+      assert_equal(new_api.test_it('my_query'), 'content')
+    end
+
+    def test_load_query_will_fall_back_to_core_queries
+      new_api = TestAPI.new(ctx: Context.new, token: 'blah', url: 'alsoblah')
+      ShopifyCli::Project.expects(:current_project_type).returns(:fake)
+
+      project_type_path = File.join(ShopifyCli::ROOT, 'lib', 'project_types', 'fake', 'graphql', 'my_query.graphql')
+      File.expects(:exist?).with(project_type_path).returns(false)
```

Comment:
Sorry for the confusion, but I don't quite understand why this returns `false`

Reply:
This test is testing that there is a current project but the file does not exist.